pkg/monitoring/metrics/virt-controller/vmistats_collector_test.go

```diff
@@ -622,6 +622,77 @@ var _ = Describe("VMI Stats Collector", func() {
 			Expect(metrics).To(BeEmpty())
 		})
 	})
+
+	Context("VMI Launcher Memory Overhead", func() {
```

Comment:
Please check that you dont emit the metric when no pod/not Running/missing compute request and on Multi-pod case you select the pod on vmi.Status.NodeName.

Reply:
@sradco Here is well I don't follow why should we handle the multiple pods case?